core/app/helpers/spree/base_helper.rb

```diff
@@ -91,7 +91,7 @@ def frontend_available?
     private
 
     def create_product_image_tag(image, product, options, style)
-      options.reverse_merge! alt: image.alt.blank? ? product.name : image.alt
+      options[:alt] = image.alt.blank? ? product.name : image.alt
```

Comment:
Hello @devilcoders 
Here `options = {:itemprop=>"image", :alt=>"<product_name>"}`, so on reverse merging the new alt value is not merged, rather old value(<product_name>) is used every time. So we can either use `merge!` method instead of `reverse_merge!` or do what I have done above.
Thanks.

Reply:
@avneetmalhotra I see, thanks for explanation.